### DISCUSSION TITLE
Get-PnPTenantSite - Inconsistent results

### DISCUSSION THREAD

**Discussion by Skierboy1** [pnp/powershell#1217] [Q&A]
Set-SPOSite -Identity "https://Contso.sharepoint.com/sites/WorkplaceToolstest" -ConditionalAccessPolicy AllowLimitedAccess

Get-PnPTenantSite https://Contso.sharepoint.com/sites/WorkplaceToolsTest | select URL, ConditionalAccessPolicy

Url                                                       ConditionalAccessPolicy
---                                                       -----------------------
https://contso.sharepoint.com/sites/WorkplaceToolsTest      AllowLimitedAccess

Get-PnPTenantSite -IncludeOneDriveSites | select URL, ConditionalAccessPolicy | where-object {$_.URL -eq 'https://Contso.sharepoint.com/sites/WorkplaceToolsTest'}

Url                                                       ConditionalAccessPolicy
---                                                       -----------------------
https://Contso.sharepoint.com/sites/WorkplaceToolsTest       AllowFullAccess

**Comment by veronicageek** [comment]
Is there a question?

**Comment by Skierboy1** [comment]
on the same object 

Get-PnPTenantSite https://Contso.sharepoint.com/sites/WorkplaceToolsTest | select URL, ConditionalAccessPolicy  returns ConditionalAccessPolicy = AllowLimitedAccess

Where as Get-PnPTenantSite -IncludeOneDriveSites | select URL, ConditionalAccessPolicy | where-object {$_.URL -eq 'https://Contso.sharepoint.com/sites/WorkplaceToolsTest'} returns ConditionalAccessPolicy = AllowFullAccess

Seems like a bug
